### ISSUE TITLE
WebAssembly

### ISSUE THREAD

**Issue by AlexReyesHuezo** [mdn/curriculum#20]
### Module number

Extension 10 WASM

### Please describe your issue

I think it would be convenient to **end** with this topic **for people with expertise** in software development.

### Describe the solution you'd like to see

It would be like a taste for an upcoming _backend_ or **JavaScript fullstack** curriculum 😉.

**Comment by chrisdavidmills** [comment]
Thanks for your thoughts @AlexReyesHuezo. I know that WebAssembly is of increasing importance, but I'm not convinced that it is a fundamental front-end web development topic. Well, it definitely wouldn't fit in Core, but in the Extensions, maybe? But even then, is this an essential fundamental front-end web dev topic? It seems to me like something too advanced to fit in the curriculum, but I am open to further thoughts on this.

**Comment by AlexReyesHuezo** [comment]
I agree with you, I only suggest it because this could go as a sample of an upcoming *backend* or *JavaScript fullstack curriculum*, if the curriculum becomes something bigger and if at some point the plans go much further.

**Comment by chrisdavidmills** [comment]
Yeah, that's a reasonable thought. There are no plans for a full-stack curriculum in the near future. But perhaps in the future. I will close this for now, but keep it in mind.
